client/jetpack-cloud/sections/pricing/header/style.scss

```diff
@@ -10,3 +10,13 @@
 		margin: 1.5em auto 0.25em;
 	}
 }
+
+/* I5 specifics */
+.is-group-jetpack-cloud.is-section-jetpack-cloud-pricing
```

Comment:
This level of specificity was required for the styles to be applied. This should be cleaned at some point.